### PR TITLE
Created Startup Script

### PR DESCRIPTION
**Pull request by mdhishaamakhtar**

### CODE REVIEW

File: wikipedia/wikipedia/spiders/index_scraper.py [hunk 1]
```diff
@@ -23,7 +22,7 @@ def parse(self, response):
         next_page_first = response.css(".mw-allpages-nav a").xpath("@href").get()
         next_page = response.css(".oo-ui-panelLayout-framed+ .mw-allpages-nav a+ a").xpath('@href').get()
         if count != 0:
-            if next_page is not None:
+            if next_page != "/w/index.php?title=Special:AllPages&from=Z%C4%83bala+%28R%C3%A2ul+Negru%29":
```
Comment (L04DB4L4NC3R) [inline wikipedia/wikipedia/spiders/index_scraper.py:25]: Store the index page in a const variable at the top

Comment (mdhishaamakhtar) [reply wikipedia/wikipedia/spiders/index_scraper.py:25]: The page is the identified ending page for the script but alright I will do that

**Comment by Geek-ubaid** [comment]
@L04DB4L4NC3R review the PR immediately.

**Comment by L04DB4L4NC3R** [comment]
Looks good. Merging

**Pull request merged**